discord/voice_client.py

```diff
@@ -207,6 +212,23 @@ def checked_add(self, attr, value, limit):
 
         self._handshake_complete.set()
 
+    @property
+    def latency(self):
```

Comment:
I don't think exposing the list is good design. It should just return the latest latency value similar to `Client.latency`.

Reply:
Changed functionality and exposure in b5f5e2b and a4a51fc.

Reply:
What happened to the `average_latency`? I thought that was fine.